dashboard/db/migrate/20200310000601_create_contact_rollups_raw.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContactRollupsRaw < ActiveRecord::Migration[5.0]
+  def change
+    create_table :contact_rollups_raw do |t|
+      t.string :email, null: false
+      t.string :sources, null: false
+      t.json :data
```

Comment:
curious how using json differs from text. We have some fields in our tables that are json but we are using text in the schema, do you know if we should be using json instead?

Reply:
I think the main reason we have big text columns instead of explicit JSON columns in our database is that we were on a older version of MySQL until a year? 2 years? ago, when we moved to 5.7, which explicitly supports JSON columns. A couple benefits listed here are validation that what's being stored is legitimate JSON, and some performance things I don't totally understand:

https://dev.mysql.com/doc/refman/5.7/en/json.html

@wjordan or @sureshc might be able to answer any more detailed questions if you have them.

Reply:
thanks! @breville something we should possibly look at? might be too late at this point

Reply:
I think we can't export JSON columns via DMS to Redshift, but we can use JSON functions on old fashioned text columns.

Reply:
mmm that sounds like a potential PITA, good to know thanks @sureshc